go/ethclient/geth_rpc_client.go

```diff
@@ -29,6 +29,7 @@ type gethRPCClient struct {
 	chainID   int                       // chainID is used to sign transactions
 	txHandler mgmtcontractlib.TxHandler // converts L1TxData to ethereum transactions
 	lock      sync.RWMutex              // ensures no concurrent tx broadcasts
+	nonce     uint64                    // current nonce tracker
```

Comment:
shouldn't the nonce be in the wallet as it is account specific?

Reply:
Yes, I was defering that to a later time, but I can easily do it now 👍